app/starship/src/main-entry.js

```
@@ -0,0 +1,178 @@
+            document.getElementById("sidebar").setAttribute("closed", "true")
+            document.getElementById("mainarea").style.opacity = 1
+        }
+    })
+    window.fullorg = org
+    var workspacelist = await authClient.organization.list()
+    var workspaceswitcherlist = ""
+    workspacelist.data.forEach(workspace => {
+        if(workspace.id == org.data.id) {
+            var active = "true"
+        } else {
+            var active = "false"
```

Biome (2.1.2)

[error] Shouldn't redeclare 'active'. Consider to delete it or rename it.

'active' is defined here:


(lint/suspicious/noRedeclare)